inputs/france/2010_stop_rep_1970-2160.xml

```diff
@@ -0,0 +1,2285 @@
+<!-- Historic, France, stops reprocessing at 2020 -->
+<simulation>
+  <control>
+    <duration>2280</duration>
+    <startmonth>1 </startmonth>
+    <startyear>1970 </startyear>
+    <decay>lazy</decay>
+  </control>
+
+  <archetypes>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Source</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Sink</name>
+    </spec>
+    <spec>
+    	<lib>cycamore</lib>
+    	<name>Reactor</name>
+    </spec>
+    <spec>
+        <lib>cycamore</lib>
+        <name>Mixer</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullRegion</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>DeployInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Enrichment</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>FuelFab</name>
+    </spec>
+  </archetypes>
+
+
+<facility>
+    <name>mixer</name>
+    <config>
+      <Mixer>
+        <in_streams>
+          <stream>
+            <info>
+              <mixing_ratio>.09</mixing_ratio>
+              <buf_size>1e+299</buf_size>
+            </info>
+            <commodities>
+              <item>
+                <commodity>Pu</commodity>
+                <pref>1.0</pref>
+              </item>
+            </commodities>
+          </stream>
+          <stream>
+            <info>
+              <mixing_ratio>.91</mixing_ratio>
+              <buf_size>1e+299</buf_size>
+            </info>
+            <commodities>
+              <item>
+                <commodity>tailings</commodity>
```

Comment:
yeah, the mixture with tailings have a closer match to the mox recipe, and that is how France creates MOX. But we can revisit this issue when the need arises!